src/styleguide/_patterns/03-organisms/01-careers/00-job-list.mustache

```diff
@@ -1,11 +1,11 @@
 <section class="job-list">
-  <h2>{{location.city}}, {{location.country}}</h2>
+  <h2>{{location.name}}, {{location.country}}</h2>
```

Comment:
see my previous comment.. you kept `city` and added `name`, so maybe it's the case of removing one of the two to avoid confusion?